python-sdk/src/astro/files/base.py

```diff
@@ -25,11 +25,13 @@ class File(LoggingMixin, Dataset):
     :param conn_id: Airflow connection ID
     :param filetype: constant to provide an explicit file type
     :param normalize_config: parameters in dict format of pandas json_normalize() function.
+    :param delta_options:
     """
 
     path: str
     conn_id: str | None = None
     filetype: constants.FileType | None = None
+    delta_options: dict = {}
```

Comment:
@dimberman as we spoke, I believe the `delta_options` we're setting relate to the transfer as opposed to the file definition - so, similar to other load operations (e.g. `native_support_kwargs`), I believe they should be part of the `load_file`.

Reply:
@tatiana fixed